docs/genome_to_collection.md

```diff
@@ -0,0 +1,235 @@
+# From KBase Genome to Collection
+
+This guide outlines the process of creating and activating a new collection from KBase narrative with Genome workspace
+objects.
+
+## Step 1: NERSC Account Setup
+The commands provided in this document are intended for execution within the terminals of 
+[NERSC Perlmutter](https://docs.nersc.gov/systems/perlmutter/). 
+If you don't have a NERSC account yet, you can create one via [Iris](https://docs.nersc.gov/iris/).
+Before proceeding, please ensure that you have an active NERSC account and are logged in to Perlmutter.
+
+```commandline
+ssh perlmutter.nersc.gov
+```
+
+## Step 2: Prepare Source Data Files (FASTA)
+
+### If you already have a KBase narrative with target Genome objects
+If you already possess a KBase narrative containing the desired Genome objects, download the FASTA files associated with
+the Genome objects from the KBase workspace to NERSC. The following command facilitates this process:
+
+```commandline
+# update arguments as needed
+workspace_id=69739
+kbase_collection=ENIGMA
+source_verion=2023.10
+env=CI
+token_filepath=path/to/token_file.txt  # This is a single-line file with the KBase token 
+
+cd /global/cfs/cdirs/kbase/collections/collections
+PYTHONPATH=. python src/loaders/workspace_downloader/workspace_downloader.py \
+    --workspace_id $workspace_id \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_verion \
+    --root_dir $root_dir \
+    --env $env \
+    --token_filepath $token_filepath \
+    --retrieve_sample 
+```
+
+### Download FASTA files from NCBI based on GTDB version
```

Comment:
does this allow you to do anything in collections? It seems like it's a dead end

Reply:
It prepares FASTA files same as what workspace downloader does. We can remove it if we are assuming collections should be created from WS only.

Reply:
You can run the pipeline and everything works?

Reply:
oh. They don't have the genome meta data. So parser won't work on them. Remove this section?

Reply:
If it's a dead end then yeah, probably should

Reply:
removed this section. I think it makes more sense to just remove it.

Reply:
:+1: